src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx

```diff
@@ -34,7 +48,11 @@ class EventOrGroupTitle extends React.Component<Props> {
 
     return subtitle ? (
       <span style={this.props.style}>
-        <GuideAnchor disabled={!hasGuideAnchor} target="issue_title" position="bottom">
+        <GuideAnchor
+          disabled={!hasGuideAnchor}
```

Comment:
we had trouble when there was multiple guide anchors with the same name but some were disabled, it worked better to render the guide anchor once instead of using disabled

Reply:
@scttcper Makes sense, what's the change you think I should make in this PR?

Reply:
i ended up doing this. Any way you want to conditionally wrap is fine though. 
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/issueList/header.tsx#L35-L45

Reply:
@scttcper got it, just don't render the anchor if it's disabled